app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @tasks = Task.all.order("created_at DESC")
+  end
+
+  def show
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def edit
+  end
```

Comment:
こちら下記のように1行で記載できます。こちらの方がメソッドの記載漏れでないことが明示できます。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod
```suggestion
  def edit; end
```